mod_payment.erl

```diff
@@ -118,8 +118,15 @@ event(#submit{message={payment, Args} }, Context) ->
                 ]},
                 Context)
     end;
-event(#submit{ message={cancel_subscription, _Args} }, Context) ->
-    UserId = z_acl:user(Context),
+event(#submit{ message={cancel_subscription, Args} }, Context) ->
+    UserId = proplists:get_value(user_id, Args, z_acl:user(Context)),
```

Comment:
Should we check for mod_payment admin usage rights when user_id =/= z_acl:user?

Reply:
Dat is wel een stukje veiliger.

Dan wordt het:

```erlang
Id =:= m.acl.user orelse z_acl:is_allowed(use, mod_payment, Context)
```

Reply:
Aangepast